packages/labs/gen-utils/src/lib/package-utils.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {promisify} from 'util';
+import {exec as execCb} from 'child_process';
+import * as fs from 'fs/promises';
+import * as path from 'path';
+import {PackageJson} from '@lit-labs/analyzer/lib/model.js';
+
+const exec = promisify(execCb);
+
+/**
+ * Map npm package name to folder in monorepo to use instead
+ */
+export interface MonorepoPackages {
+  [index: string]: string;
+}
+
+/**
+ * npm installs the package in the given packageRoot, optionally using monorepo
```

Comment:
The phrasing is a bit vague here. This is running `npm i` inside the given package root, not installing the given package root into somewhere else, right?